[2021-02-09 03:49:20,523] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: my_sample_dag.run_this2 2021-02-09T00:46:35.257642+00:00 [queued]>
[2021-02-09 03:49:20,577] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: my_sample_dag.run_this2 2021-02-09T00:46:35.257642+00:00 [queued]>
[2021-02-09 03:49:20,577] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2021-02-09 03:49:20,578] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2021-02-09 03:49:20,578] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2021-02-09 03:49:20,593] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): run_this2> on 2021-02-09T00:46:35.257642+00:00
[2021-02-09 03:49:20,598] {standard_task_runner.py:53} INFO - Started process 7498 to run task
[2021-02-09 03:49:25,737] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: my_sample_dag.run_this2 2021-02-09T00:46:35.257642+00:00 [running]> Maxims-MacBook-Air.local
[2021-02-09 03:49:30,769] {logging_mixin.py:112} INFO - hi
[2021-02-09 03:49:30,770] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-02-09 03:49:30,775] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=my_sample_dag, task_id=run_this2, execution_date=20210209T004635, start_date=20210209T004920, end_date=20210209T004930
[2021-02-09 03:49:35,753] {logging_mixin.py:112} INFO - [2021-02-09 03:49:35,751] {local_task_job.py:103} INFO - Task exited with return code 0
